scripts/integration.sh

```diff
@@ -0,0 +1,31 @@
+case $1 in
+  mac)
+    find ~/Library/Logs/Lens -type f -name *.log -delete
+  ;;
+  linux)
+    find ~/.config/Logs/Lens -type f -name *.log -delete
+  ;;
+  win)
+    find %APPDATA%/Logs/Lens -type f -name *.log -delete
+  ;;
+esac
+
+make build-extension-types build-extensions
+yarn build:$1
```

Comment:
This feels a bit odd.. could you explain why we refactored away from top-level make targets?

Reply:
Because I don't think that makefile supports what I am doing with lines 17-31

Reply:
However, some searching has found that I can do it